database/console/table_command_test.go

```diff
@@ -0,0 +1,161 @@
+				mockContext.EXPECT().Warning(mock.Anything).Run(func(message string) {
+					color.Warningln(message)
+				})
+			},
+			expected: "Table 'test' doesn't exist",
+		},
+		{
+			name: "choice table canceled",
+			setup: func() {
+				mockContext.EXPECT().NewLine()
+				mockContext.EXPECT().Option("database").Return("")
+				mockSchema.EXPECT().Connection(mock.Anything).Return(mockSchema)
+				mockContext.EXPECT().Argument(0).Return("")
+				mockSchema.EXPECT().GetTables().Return(nil, nil)
+				mockContext.EXPECT().Choice(mock.Anything, mock.Anything).Return("", assert.AnError)
+				mockContext.EXPECT().Line(mock.Anything).Run(func(message string) {
+					color.Default().Println(message)
+				})
+			},
+			expected: assert.AnError.Error(),
+		},
+		{
+			name: "get columns failed",
+			setup: func() {
+				mockContext.EXPECT().NewLine()
+				mockContext.EXPECT().Option("database").Return("")
+				mockSchema.EXPECT().Connection(mock.Anything).Return(mockSchema)
+				mockContext.EXPECT().Argument(0).Return("")
+				mockSchema.EXPECT().GetTables().Return([]schema.Table{{Name: "test"}}, nil)
+				mockContext.EXPECT().Choice(mock.Anything, mock.Anything).Return("test", nil)
+				mockSchema.EXPECT().GetColumns("test").Return(nil, assert.AnError)
+				mockContext.EXPECT().Error(mock.Anything).Run(func(message string) {
+					color.Errorln(message)
+				})
+			},
+			expected: assert.AnError.Error(),
+		},
+		{
+			name: "get indexes failed",
+			setup: func() {
+				mockContext.EXPECT().NewLine()
+				mockContext.EXPECT().Option("database").Return("")
+				mockSchema.EXPECT().Connection(mock.Anything).Return(mockSchema)
+				mockContext.EXPECT().Argument(0).Return("")
+				mockSchema.EXPECT().GetTables().Return([]schema.Table{{Name: "test"}}, nil)
+				mockContext.EXPECT().Choice(mock.Anything, mock.Anything).Return("test", nil)
+				mockSchema.EXPECT().GetColumns("test").Return(nil, nil)
+				mockSchema.EXPECT().GetIndexes("test").Return(nil, assert.AnError)
+				mockContext.EXPECT().Error(mock.Anything).Run(func(message string) {
+					color.Errorln(message)
+				})
+			},
+			expected: assert.AnError.Error(),
+		},
+		{
+			name: "get foreign keys failed",
+			setup: func() {
+				mockContext.EXPECT().NewLine()
+				mockContext.EXPECT().Option("database").Return("")
+				mockSchema.EXPECT().Connection(mock.Anything).Return(mockSchema)
+				mockContext.EXPECT().Argument(0).Return("")
+				mockSchema.EXPECT().GetTables().Return([]schema.Table{{Name: "test"}}, nil)
+				mockContext.EXPECT().Choice(mock.Anything, mock.Anything).Return("test", nil)
+				mockSchema.EXPECT().GetColumns("test").Return(nil, nil)
+				mockSchema.EXPECT().GetIndexes("test").Return(nil, nil)
+				mockSchema.EXPECT().GetForeignKeys("test").Return(nil, assert.AnError)
+				mockContext.EXPECT().Error(mock.Anything).Run(func(message string) {
+					color.Errorln(message)
+				})
+			},
+			expected: assert.AnError.Error(),
+		},
+		{
+			name: "success",
```

Comment:
No expected in this case? We can use Contains to assert it if it's a bit complicated.